src/components/panels/details/info-multiple-selection.tsx

```diff
@@ -51,16 +70,140 @@ export default function InfoMultipleSection() {
         }
     };
 
+    const [filterNodes, setFilterNodes] = React.useState(true);
+    const [filterEdges, setFilterEdges] = React.useState(true);
+    React.useEffect(() => {
+        setFilterNodes(true);
+        setFilterEdges(true);
+    }, [selected]);
+
+    const [isOpenChangeModal, setIsOpenChangeModal] = React.useState(false);
+
+    return (
+        <>
+            <Box>
+                <Heading as="h5" size="sm">
+                    {t('panel.details.multipleSelection.selected')} {selected.size}
+                </Heading>
+                <VStack m="var(--chakra-space-1)">
+                    <HStack m="var(--chakra-space-1)" width="100%">
+                        <Button
+                            size="sm"
+                            width="100%"
+                            variant={filterNodes ? 'solid' : 'ghost'}
+                            onClick={() => setFilterNodes(!filterNodes)}
+                            colorScheme="blue"
+                        >
+                            {t('panel.details.multipleSelection.filterNodes')}
+                        </Button>
+                        <Button
+                            size="sm"
+                            width="100%"
+                            variant={filterEdges ? 'solid' : 'ghost'}
+                            onClick={() => setFilterEdges(!filterEdges)}
+                            colorScheme="blue"
+                        >
+                            {t('panel.details.multipleSelection.filterEdges')}
+                        </Button>
+                    </HStack>
+                    {(filterEdges || filterNodes) && (
+                        <Button
+                            m="var(--chakra-space-1)"
+                            width="100%"
+                            size="sm"
+                            onClick={() => setIsOpenChangeModal(true)}
+                            variant="outline"
+                            colorScheme="blue"
+                        >
+                            {t('panel.details.multipleSelection.change')}
+                        </Button>
+                    )}
+                    {[...selected]
+                        .filter(id => filterNodes || !(id.startsWith('stn') || id.startsWith('misc_node')))
```

Comment:
Do we need to have misc nodes when choose filter nodes? IMO, no changes could be made on misc nodes so there is no reason to have them being selected ...?

Reply:
The color of misc-node can be changed.

Reply:
How about also displaying filtering misc-node in the UI?